lib/ansible/modules/cloud/vmware/vmware_tag_info.py

```diff
@@ -82,6 +98,31 @@
             "tag_used_by": []
         }
     }
+
+tag_info:
+    description: list of tag metadata
+    returned: on success
+    type: list
+    sample: [
+        {   "tag_name": "Sample_Tag_0002",
+            "tag_category_id": "urn:vmomi:InventoryServiceCategory:6de17f28-7694-43ec-a783-d09c141819ae:GLOBAL",
+            "tag_description": "Sample Description",
+            "tag_id": "urn:vmomi:InventoryServiceTag:a141f212-0f82-4f05-8eb3-c49647c904c5:GLOBAL",
+            "tag_used_by": []
+        },
+        {   "tag_name": "fedora_machines",
+            "tag_category_id": "urn:vmomi:InventoryServiceCategory:baa90bae-951b-4e87-af8c-be681a1ba30c:GLOBAL",
+            "tag_description": "",
+            "tag_id": "urn:vmomi:InventoryServiceTag:7d27d182-3ecd-4200-9d72-410cc6398a8a:GLOBAL",
+            "tag_used_by": []
+        },
+        {   "tag_name": "ubuntu_machines",
+            "tag_category_id": "urn:vmomi:InventoryServiceCategory:89573410-29b4-4cac-87a4-127c084f3d50:GLOBAL",
+            "tag_description": "",
+            "tag_id": "urn:vmomi:InventoryServiceTag:7f3516d5-a750-4cb9-8610-6747eb39965d:GLOBAL",
+            "tag_used_by": []
+        }
```

Comment:
would it help in this sample return to show two with the same tag names but different category ids, just to prove the usefulness of this addition?